static/sass/_pattern_careers-progression.scss

```diff
@@ -0,0 +1,379 @@
+@import "../../node_modules/vanilla-framework/scss/patterns_icons";
+
+@mixin canonical-p-careers-progression {
+  body {
+    // Fix overflown video
+    overflow-x: hidden;
```

Comment:
This will affect all body elements on this site. Is that expected? If so, I would suggest putting it somewhere central like styles.scss